assets/trix/stylesheets/attachments.scss

```diff
@@ -20,6 +20,10 @@ trix-editor {
       border-color: transparent;
     }
   }
+  
+  figure {
+    position: relative;
+  }
```

Comment:
👍 to adding this style. No need to target `figure` though, just put it in `.attachment`.